locale/fa_IR/common.xml

```diff
@@ -0,0 +1,531 @@
+    <message key="notification.removedAnnouncementType">نوع اطلاعیه حذف شد</message>
+    <message key="notification.addedAnnouncement">اطلاعیه افزوده شد</message>
+    <message key="notification.editedAnnouncement">اطلاعیه ویرایش شد</message>
+    <message key="notification.removedAnnouncement">اطلاعیه حذف شد</message>
+    <message key="notification.addedAuthor">نویسنده افزوده شد</message>
+    <message key="notification.editedAuthor">نویسنده ویرایش شد</message>
+    <message key="notification.confirmError">بروز خطا هنگام تأیید اشتراک شما</message>
+    <message key="notification.confirmSuccess">اشتراک شما فعال شد</message>
+    <message key="notification.email">جهت این‌گونه اطلاع‌رسانی برایم ایمیل بفرست</message>
+    <message key="notification.location">برو به آدرس</message>
+    <message key="notification.mailList">لیست‌ ایمیلی‌های اطلاع‌رسانی</message>
+    <message key="notification.mailList.emailInvalid">لطفاً یک ایمیل معتبر وارد کنید</message>
+    <message key="notification.mailList.privacyProtection">سیاست حفظ حریم خصوصی</message>
+    <message key="notification.mailList.protectedContent"><![CDATA[<a href="{$subscribeUrl}">اشتراک</a> برای محتوای محافظت شده.]]></message>
+    <message key="notification.mailList.submit"><![CDATA[<a href="{$submitUrl}">ارسال</a> مقاله]]></message>
+    <message key="notification.mailList.register">می‌توانید در این سایت ثبت‌نام کنید تا...</message>
+    <message key="notification.mailList.review"><![CDATA[مشارکت در روند <a href="{$reviewUrl}">داوری مقاله</a>]]></message>
+    <message key="notification.noneExist">هیچ موردی برای اطلاع‌رسانی وجود ندارد</message>
+    <message key="notification.addedNote">یادداشت پست شد</message>
+    <message key="notification.removedNote">یادداشت حذف شد</message>
+    <message key="notification.removedFile">فایل حذف شد</message>
+    <message key="notification.removedSubmission">مقاله ارسالی حذف شد.</message>
+    <message key="notification.notification">اطلاع‌رسانی</message>
+    <message key="notification.notifications">اطلاع‌رسانی‌ها</message>
+    <message key="notification.notificationsNew">({$numNew} جدید)</message>
+    <message key="notification.notificationsDescription">شما {$unreadCount} مورد اطلاعیه ناخوانده دارید و {$readCount} مورد خوانده شده.  <![CDATA[  <a href="{$settingsUrl}">Click here</a> to change your notification settings.]]></message>
+    <message key="notification.sentNotification">اطلاع‌رسانی ارسال شد</message>
+    <message key="notification.settings">تنظیمات اطلاع‌رسانی</message>
+    <message key="notification.settingsDescription">لطفاً رویدادی از سیستم را که مایلید در مورد آن به شما اطلاع‌رسانی شود انتخاب نمایید. عدم انتخاب یک رویداد سبب می‌گردد که اطلاع‌رسانی آن در فیلد اطلاع‌رسانی شما ظاهر نشود.</message>
+    <message key="notification.subscribe">مشترک شدن</message>
+    <message key="notification.subscribeError">ایمیل شما قبلاً برای ارسال اطلاع‌رسانی ثبت شده بود.</message>
```

Comment:
If this is a pain to change, don't worry about it -- but we prefer tab-based indents, not space-based indents.